mm2src/mm2_rpc/src/legacy.rs

```diff
@@ -0,0 +1,259 @@
+use common::serde_derive::{Deserialize, Serialize};
+use derive_more::Display;
+use mm2_number::{construct_detailed, BigDecimal, BigRational, Fraction, MmNumber};
+use rpc::v1::types::H256 as H256Json;
+use std::collections::HashSet;
+use std::ops::Deref;
+use uuid::Uuid;
+
+#[derive(Serialize, Deserialize)]
+pub struct KmdWalletRpcResult<T> {
```

Comment:
I would also rename `mm2_rpc` to `kmd_wallet_rpc` but it would be too much for this PR

Reply:
Related to this https://github.com/KomodoPlatform/atomicDEX-API/pull/1768#discussion_r1190473603